### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**
dependency updates

### CODE REVIEW

File: doc/changes/changes_1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# error code model java 1.0.0, released 2021-09-16
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_1.0.0.md]: Maybe use a capital case in the project name? I think it looks prettier

File: pom.xml [hunk 2]
```diff
@@ -4,15 +4,16 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.exasol</groupId>
     <artifactId>error-code-model-java</artifactId>
-    <version>0.1.0</version>
+    <version>1.0.0
+    </version>
```
Comment (AnastasiiaSergienko) [inline pom.xml]: Is it intended?

**Review by AnastasiiaSergienko — COMMENTED** [review]
Two tiny findings

File: pom.xml (hunk 2 above)
Comment (jakobbraun) [reply pom.xml]: The line break not. 1.0.0 yes. It's ready so time to switch to 1.0.0.

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**